oml/functional/metrics.py

```diff
@@ -146,13 +104,29 @@ def reduce_metrics(metrics_to_reduce: TMetricsDict) -> TMetricsDict:
     return output
 
 
-def apply_mask_to_ignore(distances: Tensor, mask_gt: Tensor, mask_to_ignore: Tensor) -> Tuple[Tensor, Tensor]:
+def take_unreduced_metrics_by_mask(metrics: TMetricsDict, mask: BoolTensor) -> TMetricsDict:
+    output: TMetricsDict = {}
+
+    for k, v in metrics.items():
+        if isinstance(v, (Tensor, np.ndarray)):
```

Comment:
Remove redundant check for `np.ndarray` due to `TMetricsDict` definition: `TMetricsDict = Dict[str, Dict[Union[int, float], Union[float, FloatTensor]]]` or change `TMetricsDict` definition

Reply:
agree, I prefer to remove the checking